AnnoyancesFilter/Popups/sections/popups_allowlist.txt

```diff
@@ -44,6 +44,11 @@ g123.jp,accounts.binance.com#@##credential_picker_container
 !
 @@||my.leadpages.net/template/$script,domain=weddingjams.com|thesaturdaypaper.com.au|careercontessa.com
 @@||my.leadpages.net^*/public/
+! https://github.com/AdguardTeam/AdguardFilters/issues/177446
+@@||v1.widget.futy.io/js/futy-widget-$script,domain=denmulderboomteelt.com
+denmulderboomteelt.com#@##cookie-notice-wrapper
+denmulderboomteelt.com#@##cookie-notice
+denmulderboomteelt.com#@##cookie-settings
```

Comment:
Is something broken when this cookie notice is hidden (I didn't notice any breakage when it's hidden, but maybe I have missed somthing)?
If not, then please add a rule to hide it, to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_specific.txt

If something is broken, then please move rules to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

Reply:
move rules as you said, cookie-notice is needed

Reply:
@AdamWr do I need to transfer it to the top without comment with link to task?

Reply:
We usually add new rules at the top, somewhere here - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt#L17-L20
so please move it there, but comment with link to issue should be added.

By the way, could you please write what exactly is broken when cookie notice is hidden?

Reply:
And one more question, why this exception rule:
```adblock
@@||v1.widget.futy.io/js/futy-widget-$script,domain=denmulderboomteelt.com
```
is required?
It's not blocked on my end, but maybe I have missed something.

Reply:
you are right, it is unnecassary

Reply:
`By the way, could you please write what exactly is broken when cookie notice is hidden?
`
Cookie popup is not opening without this rule

Reply:
So, maybe it can be just hidden, for example:
```adblock
denmulderboomteelt.com###cookie-notice-wrapper
```

Reply:
@AdamWr not working like that
just in combination, that into the PR

moved rules to the top

Reply:
I mean just this rule:
```adblock
denmulderboomteelt.com###cookie-notice-wrapper
```
without adding exceptions.

Reply:
@AdamWr In that case the cookie banner is not visible at all, including the background.
Is this good?

Reply:
The main purpose of AdGuard Cookie Notices filter/AdGuard Annoyances filter is to hide/block these cookie notices/popups.
We unblock them only if website functionality is broken when cookies are not accepted/rejected.
If there is no breakage when cookie notice is hidden then it's preferred solution.

Reply:
Ok, done as you said, all works fine
@AdamWr

Reply:
Sorry, I should mention it, but it should be added to `cookies_specific.txt`.
There is already similar rule, so please add a domain to it:
https://github.com/AdguardTeam/AdguardFilters/blob/ff067d646e6ff9f9ba2451745dd187f05a30c29f/AnnoyancesFilter/Cookies/sections/cookies_specific.txt#L8064
and it this case adding a comment is not necessary.

Reply:
@AdamWr done as you said